elements/lisk-cryptography/src/constants.ts

```diff
@@ -15,11 +15,12 @@
 export const SIGNED_MESSAGE_PREFIX = 'Lisk Signed Message:\n';
 export const BINARY_ADDRESS_LENGTH = 20;
 export const DEFAULT_LISK32_ADDRESS_PREFIX = 'lsk';
-export const DEFAULT_BASE32_ADDRESS_PREFIX = DEFAULT_LISK32_ADDRESS_PREFIX;
 export const ED25519_CURVE = 'ed25519 seed';
 export const MAX_UINT32 = 4294967295;
 export const HARDENED_OFFSET = 0x80000000;
 export const HASH_LENGTH = 32;
 export const L = 8160; // 32 * 255
 export const EMPTY_SALT = Buffer.from(new Array(32).fill(0));
 export const SHA256 = 'sha256';
+export const BASE32_CHARSET = 'zxvcpmbn3465o978uyrtkqew2adsjhfg';
```

Comment:
We don't need to expose this charset

Reply:
The reason I moved it to `constants.ts` and exported it is to be able to use it in tests.